### DISCUSSION TITLE
Welcome to PocketSDR Discussions!

### DISCUSSION THREAD

**Discussion by tomojitakasu** [tomojitakasu/PocketSDR#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by iceberg1369** [comment]
Hello all,
I am jeff from IRAN, focused on rtk receivers design.  I personally love rtklib. 
For the pocketsdr.
There is a pretty well eatablished gnss-sdr project out there. But gnss-sdr project lacks hardware part. I think its a quite good idea to offload heavy lifting of acquisition and tracking stage to fpga and make the project a realtime receiver. Another field could be null steering . But I think 2bit adc is not enouth for this.
